Cesium.Compiler/Compilation.cs

```diff
@@ -17,12 +17,15 @@ internal static class Compilation
         IEnumerable<string> inputFilePaths,
         string outputFilePath,
         TargetRuntimeDescriptor targetRuntime,
-        ModuleKind? moduleKind = null,
-        string @namespace = "",
-        string globalClassFqn = "")
+        ModuleKind? moduleKind,
+        string corelibAssembly,
+        string cesiumRuntime,
+        IList<string> defaultImportAssemblies,
+        string @namespace,
+        string globalClassFqn)
```

Comment:
Okay, it quickly becomes a lot of parameters. Let's move them into a `record`? What do you think?